cli/pcluster/cw-dashboard-substack.cfn.yaml

```diff
@@ -192,6 +193,27 @@ Resources:
               {%- endfor %}
               - !Sub '],"region":"${AWS::Region}"}}'
 
+              # EBS VolumeThroughputPercentage graph
+              {% set firstmodule = {'bool': True} %} # TODO erase dictionary needed to be able to change the value inside the loop
```

Comment:
I'd rename firstmodule to first_metric, but let's see if we can get rid of it.

Reply:
Done renaming, will see if we can get rid of it